packages/containers-shared/src/knobs.ts

```diff
@@ -2,3 +2,13 @@
 export const getCloudflareContainerRegistry = () => {
 	return process.env.CLOUDFLARE_CONTAINER_REGISTRY ?? "registry.cloudflare.com";
 };
+
+/**
+ * Given a container image that is a registry link, this function
+ * returns true if the link points the Cloudflare container registry
+ * (defined as per `getCloudflareContainerRegistry` above)
+ */
+export function isCloudflareRegistryLink(image: string) {
+	const cfRegistry = getCloudflareContainerRegistry();
```

Comment:
This will create a new string everytime it gets executed. Using something like this and definining cfRegistry in global scope will remove the unnecessary string allocation.

```suggestion
	cfRegistry ??= getCloudflareContainerRegistry();
```